### ISSUE TITLE
Suggestion: Group meetings

### ISSUE THREAD

**Issue by archthegit** [CS2103AUG2017-T09-B2/main#184]
Seeing as your application is centred around people who have a lot of meetings, maybe you could enable group meetings with more than one contact

**Comment by kyngyi** [comment]
Yes we have group meetings, we can add meetings with diff people for same location, date and meeting name

**Comment by archthegit** [comment]
Yes but it is not directly evident that it is enabling group meetings, it might be better to have one meeting card with multiple contacts' names
